src/main/java/network/brightspots/rcv/FileUtils.java

```diff
@@ -50,6 +56,29 @@ static void createOutputDirectory(String dir) throws UnableToCreateDirectoryExce
     }
   }
 
+  static String getHash(File file) {
+    MessageDigest digest;
+    try {
+      digest = MessageDigest.getInstance("SHA-256");
+    } catch (NoSuchAlgorithmException e) {
+      Logger.severe("Failed to get SHA-256 algorithm");
+      return "[hash not available]";
+    }
+
+    try (InputStream is = Files.newInputStream(file.toPath())) {
+      try (DigestInputStream hashingStream = new DigestInputStream(is, digest)) {
+        while (hashingStream.readNBytes(1024).length > 0) {
+          // Read in 1kb chunks -- don't need to do anything in the body here
+        }
+      }
+    } catch (IOException e) {
+      Logger.severe("Failed to read file: %s", file.getAbsolutePath());
+      return "[hash not available]";
+    }
+
+    return Utils.bytesToHex(digest.digest());
+  }
```

Comment:
Oh, cool. So this means that the footprint is O(1) instead of O(n)? Also curious why you chose 1024 instead of, say, that number squared or whatever. Not sure it makes any meaningful difference in perf.

Reply:
I assume it doesn't affect performance much either way, though we can revisit if we end up hashing large CVRs and performance becomes a concern.